ginga/Control.py

```diff
@@ -850,16 +866,22 @@ def add_image(self, imname, image, chname=None, silent=False):
         if chname is None:
             channel = self.get_channelInfo()
             if channel is None:
-                raise ValueError("Need to provide a channel name to add the image")
+                raise ValueError("Need to provide a channel name to add "
+                                 "the image")
             chname = channel.name
 
+        # Initialize MODIFIED header keyword for image-modified callback.
+        # This is for consistent keyword display in Contents, and used in
+        # 'image-modified' callback.
+        image.metadata['header']['MODIFIED'] = None
```

Comment:
First, if this is being added to the FITS header, it would be better to use `image.set_keyword()` which doesn't expose the implementation detail that the FITS header is stored in the metadata "header" attribute--which could change in the future.

Second, this will make MODIFIED be shown in the `Header` plugin.  Is this desirable?  Because it will make users think that it is actually part of the FITS header.

Third, because of #1 and #2, I think it might be better to make 'modified' be a regular metadata attribute that is assigned in the `AstroImage` class (or even `BaseImage`) as `None`.  And it should be updated (set to a `time.time()`) when the `set_data()` method is called in that class.  That class already invokes a `modified` callback.